cacheops/conf.py

```diff
@@ -38,7 +39,7 @@ def _setup(self):
         except AttributeError:
             raise ImproperlyConfigured('You must specify CACHEOPS_REDIS setting to use cacheops')
 
-        client = (SafeRedis if DEGRADE_ON_FAILURE else redis.StrictRedis)(**redis_conf)
+        client = (SafeRedis if DEGRADE_ON_FAILURE else LocalCachedTransactionRedis)(**redis_conf)
```

Comment:
> If DEGRADE_ON_FAILURE is True then transactions are not supported.

Picking this up after a little less then a week. My understanding of DEGRADE_ON_FAILURE is that `True` means turn redis.ConnectionError and redis.TimeoutError into warnings. `False` would mean these exceptions bubble to the top. Am missing some knowledge on why DEGRADE_ON_FAILURE's value would effect transactions?